iceoryx_posh/source/runtime/posh_runtime.cpp

```diff
@@ -88,26 +97,26 @@ PoshRuntime::~PoshRuntime() noexcept
 }
 
 
-const ProcessName_t& PoshRuntime::verifyInstanceName(const ProcessName_t& name) noexcept
+const ProcessName_t& PoshRuntime::verifyInstanceName(cxx::optional<const ProcessName_t*> name) noexcept
 {
-    if (name.empty())
+    if (!name.has_value())
```

Comment:
on a second thought, maybe it makes more sense to do this check in `PoshRuntime& PoshRuntime::getInstance(cxx::optional<const ProcessName_t*> name)`, right before calling `return getRuntimeFactory()(name);` and then passing just a reference to the name instead the optional